csi-cluster-driver-registrar/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM golang:1.13.5 AS build
+ENV GIT_UPSTREAM_REPO=https://github.com/kubernetes-csi/cluster-driver-registrar
+ENV GIT_VERSION=v1.0.1
+WORKDIR /go/src/github.com/storageos/images/csi-cluster-driver-registrar
+COPY LICENSE .
+COPY csi-cluster-driver-registrar/build.sh /
+
+# Clone upstream with full history and checkout to the forked branch to retain
+# correct version tag used in the build flag.
+RUN git clone $GIT_UPSTREAM_REPO $GOPATH/src/github.com/kubernetes-csi/cluster-driver-registrar
+RUN cd "$GOPATH/src/github.com/kubernetes-csi/cluster-driver-registrar" \
+    && git checkout $GIT_VERSION \
+    && bash /build.sh
+    # The default make target builds linux and windows binaries together. Use
+    # make when the build server has enough resources.
+    # && make
```

Comment:
We should be able to use make directly https://github.com/kubernetes-csi/cluster-driver-registrar/blob/v1.0.1/Makefile#L35 , linux only build. `build.sh` can be removed.

Reply:
Yes, tested with just `make` and it works.